plugins/hydra_joblib_launcher/hydra_plugins/hydra_joblib_launcher/config.py

```diff
@@ -25,7 +25,7 @@ class JobLibLauncherConf:
     verbose: int = 0
 
     # timeout limit for each task
-    timeout: Optional[int] = None
+    timeout: Optional[float] = None
```

Comment:
1. What is the unit of the timeout? can you add a comment?
2. Did you verify in the joblib docs that it can take a float as the timeout?

Reply:
So, can you add a comment with the time units?
If this is specific to loki, just mention it in the comment.